src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -29,13 +29,12 @@ public static function check_for_updates() {
 	 * @since 5.5.0 Added pretty permalinks support information.
 	 *
 	 * @throws ImagickException
-	 * @global wpdb  $wpdb               WordPress database abstraction object.
```

Comment:
nice! I know this is just moving it around, but it's nice seeing it leave the bigger general function and become isolated in the DB-related function.